book/content/diving_into_leadership/diving_into_leadership.md

```diff
@@ -0,0 +1,49 @@
+# Personal Experience: Diving into Leadership
+
+_The following blog post was written by The Turing Way contributor [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson)._
+
+Hackathons can be quite intimidating scenarios to put yourself into.
+You're probably meeting a whole load of new people and getting to grips with a new project or programming language.
+On top of that, you probably only have a couple of days or less to produce something to show for your time!
+
+Now imagine if it was your job to lead and coordinate that group of people to work on your own project?
+
+Don't worry! I've been there, and I was intimidated too!
```

Comment:
```suggestion
Yeah! That is challenging for many! I've been there, and I was intimidated too!
```

Reply:
You don't need to accept this. I just wanted to remove "don't worry" as that's never helpful! 😅